### ISSUE TITLE
Is it possible to persist my code into a database?

### ISSUE THREAD

**Issue by syang** [microsoft/pxt#9393]
I am thinking of building a local community where kids can share their code with each other.

I understand that Makecode does not support Cloud (or self-hosted backend) persistence. Can this be done? 

I am not super familiar with the architecture of makecode, but if this is something aligned with the community, I’d love to be guided to contribute to this idea (if the core team does not have the bandwidth to work on it)

**Comment by riknoll** [comment]
@syang which of our editors are you looking for this feature in?

We have accounts with sign-in on pxt-arcade

**Comment by syang** [comment]
@riknoll I am thinking of Makecode for micro:bit. Essentially I’d like self-host a makecode for my site, and allowing kids to save their work there.

can you elaborate what you meant by “we have accounts with sign-in Oct-arcade”?

**Comment by Jaqster** [comment]
Jumping in here - we do support cloud synch of student projects for MakeCode Arcade (https://makecode.com/blog/arcade/intro-cloud-sync).  And we are working on enabling this with micro:bit soon!

**Comment by abchatra** [comment]
Yes, we plan to add this support to micro:bit soon. @eanders-ms  FYI
